dev-resources/template/2_lrs.yml

```diff
@@ -50,10 +50,12 @@ Parameters:
   # Lambda Script
   S3Bucket:
     Type: String
-    Default: lrsql-demo
+    Description: S3 Bucket used to source the DB Init function. The default is hosted publicly so you probably don't need to change this.
+    Default: lrsql-dbfn
   S3Key:
     Type: String
-    Default: db-init.zip
+    Description: Override for S3 key to source the DB Init function. Drawn from LrsVersion by default. You probably don't want to change this.
+    Default: ""
```

Comment:
You warn against changing this, yet you allow it and even accommodate for it later in the template. Curious. (In all seriousness I imagine people _would_ want to override this if they have custom init functions.)

Reply:
Yeah, custom init functions OR if Yet ceased to exist (and thus the bucket) and someone still wanted to deploy

Reply:
Cool, I just thought the wording "You probably don't want to change this" felt a bit too strong.

Reply:
But they probably don't

Reply:
Ok if you say so